README.md

```diff
@@ -34,7 +34,7 @@ For whole slide image formats based on the tiff standard, `imagedephi` allows a
 
 
 ## Ruleset Format Overview
-In order to read the base rules and build your own custom rule sets, it is important to understand the format in which rulesets are specified. Rulesets are defined by `.yaml` files (one ruleset per file), and are a dictionary with the following top-level tags: `name`, `description`, `output_file_name`, `tiff`, and `svs`.
+In order to read the base rules and build your own custom rule sets, it is important to understand the format in which rulesets are specified. Rulesets are defined by `.yaml` files (one ruleset per file), and are a dictionary with the following top-level tags: `name`, `description`, `output_file_name`, `tiff`, `svs` and `dicom`.
```

Comment:
We've lost the Oxford comma.  Elsewhere we use the Oxford comma.

Reply:
Fixed in 8101650